### PR TITLE
feat: LT-990 Case SLA

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**
Create SLA circles with colours & percentage representing different information about the SLA

Screenshot 2020-02-24 at 10 58 55

Also tidy up view application tests into 3 feature files: Open, Standard & MOD

### CODE REVIEW

File: ui_automation_tests/features/assign_users_to_queue.feature [hunk 1]
```diff
@@ -4,38 +4,37 @@ Feature: I want to assign one or more specific users to a case in a work queue
...
-  Background:
-    Given I sign in to SSO or am signed into SSO
-    And I create open application or open application has been previously created
-    When I go to queues
-    When I enter in queue name Review
-    And I go to application previously created
-    And I add case to newly created queue
-    And I go to the internal homepage
-    And I click on the added queue in dropdown
+#  Background:
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/assign_users_to_queue.feature]: @phlptt requested I comment out these tests due to a bug in affecting UAT

File: ui_automation_tests/features/case_officer.feature [hunk 2]
```diff
@@ -5,15 +5,15 @@ Feature: I want to assign a dedicated case officer / single point of contact to
...
-    And I click Assign Case Officer Button
-    And filter by test user name
-    Then I should see one user with the test user name
-    When I click the user and assign
-    And I click Assign Case Officer Button
-    Then I see a case officer is assigned
-    When I click unassign
-    And I click Assign Case Officer Button
-    Then I see no case officer is assigned
+#    Given I sign in to SSO or am signed into SSO
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/case_officer.feature]: @phlptt requested I comment out these tests due to a bug in affecting UAT

File: ui_automation_tests/conftest.py [hunk 3]
```diff
@@ -326,3 +329,18 @@ def create_clc_query(driver, apply_for_clc_query, context):
 def filter_status_change(driver, context, status):
     CaseListPage(driver).select_filter_status_from_dropdown(status)
     CaseListPage(driver).click_apply_filters_button()
+
+
+@then("I should see my case in the cases list")  # noqa
+def case_in_cases_list(driver, context, internal_url):
+    driver.get(internal_url.rstrip("/") + "/cases/")
```
Comment (phlptt) [inline ui_automation_tests/conftest.py]: This should be split out into two steps
When
do an action
Then
assert the result of an action

File: ui_automation_tests/features/view_standard_application.feature [hunk 4]
```diff
@@ -8,17 +8,13 @@ Feature: I want to view the case details of a case
...
     And I see an ultimate end user
     And I see a third party
     And I see a consignee
-
-  @LT_1658_parties_refactor @regression
-  Scenario: Gov user can see inactive parties on case
-    Given I sign in to SSO or am signed into SSO
-    And I create standard application or standard application has been previously created
-    And the exporter has deleted the third party
+    Given the exporter has deleted the third party
```
Comment (phlptt) [inline ui_automation_tests/features/view_standard_application.feature]: Does this work? Usually it doesnt allow a Given preceded by Whens and Thens

File: ui_automation_tests/conftest.py (hunk 3 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/conftest.py]: Ok

File: ui_automation_tests/features/view_standard_application.feature (hunk 4 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/features/view_standard_application.feature]: Yep :)

File: ui_automation_tests/conftest.py (hunk 3 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/conftest.py]: Done

File: ui_automation_tests/features/view_open_application.feature [hunk 5]
```diff
@@ -0,0 +1,11 @@
+@all @MOD
```
Comment (stephanieboyd) [inline ui_automation_tests/features/view_open_application.feature]: Surely not MOD tag here?

Comment (BenAndrew-Kainos) [reply ui_automation_tests/features/view_open_application.feature]: Fixed

**Pull request merged**